src/main/kotlin/uk/gov/dluhc/emsintegrationapi/rest/PostalVoteApplicationController.kt

```diff
@@ -0,0 +1,41 @@
+package uk.gov.dluhc.emsintegrationapi.rest
+
+import mu.KotlinLogging
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.security.core.Authentication
+import org.springframework.web.bind.annotation.CrossOrigin
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.dluhc.emsintegrationapi.constants.ApplicationConstants.Companion.PAGE_SIZE_PARAM
+import uk.gov.dluhc.emsintegrationapi.models.PostalVoteAcceptedResponse
+import uk.gov.dluhc.emsintegrationapi.rest.PostalVoteApplicationController.Companion.ROOT_PATH
+import uk.gov.dluhc.emsintegrationapi.service.PostalVoteApplicationService
+
+private val logger = KotlinLogging.logger { }
+
+@RestController
+@CrossOrigin
+@RequestMapping(ROOT_PATH)
+class PostalVoteApplicationController(private val postalVoteApplicationService: PostalVoteApplicationService) {
+    companion object {
+        const val ROOT_PATH = "/postalVotes"
+        const val ACCEPTED = "/accepted"
```

Comment:
just thinking is it worth making constants for these if only referenced once in prod code? (the ref by tests should prob use the fixed value rather than ref this const to protect against unexpected changes)

Reply:
By having here any path changes automatically reflect on the tests. Otherwise I am not sure about the unexpected changes.